tests/tests/test_ft_face.py

```diff
@@ -1,6 +1,11 @@
+import os
+
 import freetype
 
 
 def test_ft_face():
     """A smoke test."""
     assert freetype.Face('../examples/Vera.ttf')
+
+    if os.environ.get("FREETYPEPY_BUNDLE_FT"):
+        assert freetype.version() == (2, 9, 0)
```

Comment:
then you'll need to remember to update this test every time your bump the bundled freetype shared library

Reply:
Hm. You mean I should scrape this from setup-build-freetype.py? :grin:

Reply:
yeah, you could do that to DRY

hey I just noticed, why do you have two nested `tests/tests/` subfolders?

Reply:
Because @rougier was against placing freetype inside a Lib/ folder ;)

Reply:
that's horrible. you don't need that. pytest can just find the tests wherever they are in current folder and all its subfolders...

Reply:
Yes, but I want tox-safeness :syringe:

Reply:
I still don't get it. what's the problem with cd'ing into the tests directory (where .py test files are) and run pytest from there?

Reply:
Dunno, but if that ensures that the installed package is toxed, okay :grin:

Reply:
but you've already set `changedir=tests` here in tox.ini, no?
https://github.com/madig/freetype-py/blob/d46e6347a9b9c782de8631d5d6d936bfbb9d7ab8/tox.ini#L7

that should already ensure that the pytest command is executed with a current working directory set to "tests", and thus free from the stuff which is in the root of the repo (setup.py, the freetype package source itself, etc.)

Reply:
I just tried locally to `git mv tests/tests/test_ft_face.py tests/test_ft_face.py` and run `FREETYPEPY_BUNDLE_FT=1 tox` and it just worked.

By the way, I use [`tox-venv`](https://github.com/tox-dev/tox-venv) instead of plain `tox` on my machine. Tox-venv uses the built-in `venv` module (python3 only) whereas `tox` still uses the `virtualenv` module (with all its hacks to make it work for both 2 and 3), though they are discussing about making tox-venv the default for python3.

Anyway, when you `python3 -m venv somenthing`, it does not automatically install the `wheel` package for your, unlike the old virtualenv. Your setup.py raise an import error when attempting to build the embedded shared library if wheel is not installed in the current environment.
You should probably add `wheel` in the `deps=` list inside tox.ini, or make it optional in setup.py.

Reply:
Brain lapse, will implement :)

Tox runs both py27 and py36 tests fine, but will add wheel.

Now for how to scrape the version information...
```
    def test_bundled_version():
        if os.environ.get("FREETYPEPY_BUNDLE_FT"):
>           from .. import setup_build_freetype
E           ValueError: Attempted relative import in non-package
```

Reply:
Don’t import it, open the file and regex it